build/scripts/install-build-deps.sh

```diff
@@ -9,6 +9,8 @@ fi
 
 if [ "$(uname)" = "Darwin" ] ; then
   OSARCH="darwin-amd64"
+elif [ "$(uname -i)" = "aarch64" ] ; then
```

Comment:
Understand this is existing logic, however, I am curious why these two parameters are requried for `go install`. Will go env GOARCH and GOOS be assgined automatically ?

Reply:
GOOS and GOARCH _should_ be assigned based on build host architecture but it seems the "OSARCH" env var is used elsewhere, as I can build successfully on arm64 (Graviton2) with it, but not without it.

Reply:
I don't have the exact error in my notes, let me try again and document it.

Reply:
yup, I tried to search OSARCH in repo, but can't find any place using this variable.